sys-apps/openrazer/openrazer-3.4.0-r3.ebuild

```diff
@@ -48,7 +48,11 @@ DOCS=( README.md )
 
 DOC_CONTENTS="To successfully use OpenRazer: load desired kernel module
 (razeraccessory, razerkbd, razerkraken and/or razermouse),
-add Your user to the \"plugdev\" group and start the \"openrazer-daemon\"."
+add your user to the \"plugdev\" group and start the OpenRazer daemon.
+\n\n
```

Comment:
this will disappear because autoformatting is turned on (see readme.gentoo-r1.eclass docs)

imho, lets go with autoformatting and remove those two newlines

Reply:
I just figured if you install it the first time around you maybe miss it in the console log.

Reply:
> I just figured if you install it the first time around you maybe miss it in the console log.

yea... tho this is a unstable package plus I'm working on a wiki page for it and all important info like this one  should be mirrored there

https://wiki.gentoo.org/wiki/User:Xgqt/OpenRazer